### ISSUE TITLE
version 2 is not working (for me at least)

### ISSUE THREAD

**Issue by thunfisch987** [victorgarciaesgi/nuxt-typed-router#57]
_steps to reproduce:_

- create a new nuxt 3 project (with `nuxi init`)
- `npm install`
- `npm install -D nuxt-typed-router` (as in the readme)
- or
- `npm install nuxt-typed-router` (as on the new site)
- add nuxt-typed-router to nuxt.config.ts:
  - export default defineNuxtConfig({
        modules: ["nuxt-typed-router"],
    });
- run `npm install` or `npm nuxt prepare`

```
 ERROR  The URL must be of scheme file

  at new NodeError (node:internal/errors:399:5)
  at fileURLToPath (node:internal/url:1492:11)
  at setup ([path to project]/node_modules/nuxt-typed-router/dist/module.mjs:647:24)
  at normalizedModule ([path to project]/node_modules/@nuxt/kit/dist/index.mjs:167:29)
  at async installModule ([path to project]/node_modules/@nuxt/kit/dist/index.mjs:416:3)
  at async initNuxt ([path to project]/node_modules/nuxt/dist/index.mjs:1825:7)
  at async loadNuxt ([path to project]/node_modules/nuxt/dist/index.mjs:1857:5)
  at async loadNuxt ([path to project]/node_modules/@nuxt/kit/dist/index.mjs:493:19)
  at async Object.invoke ([path to project]/node_modules/nuxi/dist/chunks/prepare.mjs:30:18)
  at async _main ([path to project]/node_modules/nuxi/dist/cli.mjs:50:20)
```

_environment stuff:_

- node/npm version: 19.4.0 / 19.3.0 (tested on both)
- nuxt version: 3
- nuxt-typed-router version: 2.0.0 / 2.0.1

_additional comments_
it's perfectly working with versions below 2 (tested with 1.2.5)

i'm also confused that the release of 2.0.0 here on github was 2 days ago and the last commit was 2 days ago, but there's a version 2.0.1 on npm (not on github) that was released ~7/8 hours ago

**Comment by thunfisch987** [comment]
npm install -D nuxt-typed-router (as in the readme.md)
and
npm install nuxt-typed-router (as on the new site)
makes no difference

**Comment by thunfisch987** [comment]
found the error, tested a fix, gonna do a PR

**Comment by victorgarciaesgi** [comment]
Hi Thanks for the report! 

Weird error, is it because of the node version? 

Yeah sorry for the version confusion, I had to publish in hurry the `2.0.1` because of a breaking bug, I didn't had time to make a release for it 😅

**Comment by victorgarciaesgi** [comment]
Thanks for your PR! Didn't know about `createResolver`, merging it right away

**Comment by victorgarciaesgi** [comment]
Released in `2.0.2`, thanks again @thunfisch987 ! :D
